README.md

```diff
@@ -1,254 +1,149 @@
-<table>
-  <tr>
-    <td valign="center"><a href="https://github.com/bebasid/bebasit/blob/master/docs/openwrt-tutorial.md"><img src="https://archive.org/download/github.com-openwrt-openwrt_-_2021-04-13_21-34-02/cover.jpg" width="24"/> OpenWRT</td>
-    <td valign="center"><a href="https://github.com/bebasid/bebasit/blob/master/docs/mikrotik-tutorial.md"><img src="https://cdn.discordapp.com/attachments/381569782257811456/1147154206063611994/Sb0qyQ88hIP9IdDM.png" width="16"/> MikroTik</td>
-</table></br>
-        <b>> Sistem Operasi (OS)</b>:</br>
-<table>
-  <tr>
-    <td valign="center"><a href="https://github.com/bebasid/bebasit/blob/master/README.md#windows"><img src="https://seeklogo.com/images/W/windows-11-icon-logo-6C39629E45-seeklogo.com.png" width="24"/> Windows</td>
-    <td valign="center"><a href="https://github.com/bebasid/bebasit/blob/master/README.md#macos"><img src="https://cdn-icons-png.flaticon.com/512/2/2235.png" width="24"/> MacOS</td>
-    <td valign="center"><a href="https://github.com/bebasid/bebasit/blob/master/README.md#linux"><img src="https://cdn-icons-png.flaticon.com/512/226/226772.png" width="24"/> Linux</td>
-</table></br>
-</div>
-<hr>
-<div align="center">
-    <b><code><a href=#bebasid>BebasID</a> ini telah disponsori oleh:</code></b><br><br>
-  <table align="center">
-<thead>
-<tr>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td>
-    <img src="https://github.com/bebasid/bebasdns/blob/main/dev/resources/logo/sponsor/atharva.svg#gh-light-mode-only" width="330" target="_blank" alt="Website Provider Atharva">
-    <img src="https://github.com/bebasid/bebasdns/blob/main/dev/resources/logo/sponsor/atharva_white.svg#gh-dark-mode-only" width="330" target="_blank" alt="Website Provider Atharva">
-  </td>
-<td>
-    <img src="https://github.com/bebasid/bebasid/blob/master/dev/resources/logo-atharva-cloud-black.png#gh-light-mode-only" width="330" target="_blank" alt="Website Provider Cloud Atharva">
-    <img src="https://github.com/bebasid/bebasid/blob/master/dev/resources/logo-atharva-cloud-whitepng.png?raw=true#gh-dark-mode-only" width="330" target="_blank" alt="Website Provider Cloud Atharva">
-</td><tr>
-  <th>https://www.atharva.co.id/</th>
-<th>https://www.atharva.cloud</th>
-</tr>
-</tbody>
-</table>
-    <i>Penyedia VPS yang berorientasi pada kebebasan dan netral untuk kebutuhannya.</i></br>
-    <m>Dimulai dari <b>Rp. 50.000,00</b></m>.</br>
-    <a href="https://www.atharva.cloud/index.php?rp=/store/cloud-vm-lite">Lihat paket langganan VPS selengkapnya</a>.
-</div>
-<hr>
-<p align="center"><code>KINI Memperkenalkan:</code></br></br>
-    <img src="https://github.com/bebasid/bebasid/blob/master/dev/resources/logo-black.svg#gh-light-mode-only" alt="bebasID White Logo" width="445">
-    <img src="https://github.com/bebasid/bebasid/blob/master/dev/resources/logo.svg#gh-dark-mode-only" alt="bebasID Black Logo" width="445"></br>
-    <i>Membantumu berselancar dengan bebas dan tak ada habisnya!</i>
-</p>
+#### Bahasa / Language
```

Comment:
Add the tables for this title, make it to group

Reply:
I dont think it would add anything to the look, and it would make the raw file look more cluttered than it already is